### PR TITLE
ci: 466 initiate btp node for archway on icon testnet

### PR DESCRIPTION
**Pull request by DeepakBomjan**
### Commit Message

```bash
type: commit message
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

### Changelog Entry

```bash
version: <log entry>
```

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [ ] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

**Comment by DeepakBomjan** [comment]
Hi @AntonAndell @redlarva @nightowl121 , I have added the workflow to deploy javascore contracts on Berlin testnet, I have updated the score address in gradle.properties file. The contracts which failed to deploy have been commented for now. Please have a look on failed contracts.

File: contracts/javascore/gradle.properties [hunk 1]
```diff
@@ -31,3 +31,14 @@ uat.contracts.xcall-multi-protocol=cxe31140b7cc953779fe33681020e08f9d0d158757
 local.contracts.ibc-core=hxb6b5791be0b5ef67063b3c10b840fb81514db2fd
 local.contracts.xcall=hxb6b5791be0b5ef67063b3c10b840fb81514db2fd
 local.contracts.xcall-multi-protocol=hxb6b5791be0b5ef67063b3c10b840fb81514db2fd
+
+#Berlin
+berlin.contracts.ibc-core=cxfb7563611a92349d53f60fadad4f90291490eeca
+berlin.contracts.tendermint=cxa2a78e9f53d4e5f1f2b0994801a0daa8cc6fcc8b
+berlin.contracts.mock-app=cx3b7728befe3a6049fed4382bf2ec70c6494ec396
+berlin.contracts.mock-client=cx28bea26290f009b8a4ee274de48fc84e1a83631c
+# berlin.contracts.mock-dapp=cx6a52b5f7e24c2bcea7758341dcb599f18dd03ecb
```
Comment (AntonAndell) [inline contracts/javascore/gradle.properties:40]: Can be removed, since contract wont be used

File: contracts/javascore/gradle.properties [hunk 2]
```diff
@@ -31,3 +31,14 @@ uat.contracts.xcall-multi-protocol=cxe31140b7cc953779fe33681020e08f9d0d158757
...
+berlin.contracts.mock-dapp-multi-protocol=cx7cb9504e8bd159fff52e6bf7f7e205b381b6191a
+berlin.contracts.xcall=cx5a043c58a278cd84fea67d36d4f45b44775eeaf3
```
Comment (AntonAndell) [inline contracts/javascore/gradle.properties:42]: Can be removed, since contract wont be used

**Review by AntonAndell — CHANGES_REQUESTED** [review]
Can you also add a section to the readme. Like this: 
https://github.com/iconloop/btp2-testnet/blob/master/README.md

**Comment by DeepakBomjan** [comment]
Opening new PR as the current PR carries an irrelevant changes also

**Pull request closed without merging**